src/go/rpk/pkg/utils/encoding.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	bytes2 "bytes"
+	"encoding/binary"
+)
+
+func WriteString(bytes []byte, string string) ([]byte, error) {
```

Comment:
Are these functions used from more than 1 place?

Reply:
Also if there's a collision between the name of a var and a package, it's clearer to just rename the var. I've seen `bs` used for `[]byte`. Same for `string string`: name it `s` or `str`.

Reply:
> Are these functions used from more than 1 place?

Yes

Reply:
> Also if there's a collision between the name of a var and a package, it's clearer to just rename the var. I've seen `bs` used for `[]byte`. Same for `string string`: name it `s` or `str`.

Done